### ISSUE TITLE
Yggdrasil compatibility with CJDNS - ie running Yggdrasil only networks

### ISSUE THREAD

**Issue by makew0rld** [tomeshnet/prototype-cjdns-pi#244]
I think there should eventually be the ability to run Yggdrasil only networks. This would require making CJDNS installation not mandatory, and providing yggdrasil node addresses in places where CJDNS ones exist, such as for NTP, DNS and IPFS peering.

- [ ] NTP - exists on the network, known person running it
- [ ] DNS - exists on the network, known person running it
- [x] IPFS peering
- [x] SSB peering
- [x] Making CJDNS a optional module (#320)

**Comment by darkdrgn2k** [comment]
First some one needs to run perminent services on the network.

**Comment by makeworld-the-better-one** [comment]
Yes, of course. Are any of your current servers that run some of these services running ygg? Could they at some point? I would also tag @Famicoman because he runs some of the DNS.

**Comment by benhylau** [comment]
I think the default/basic install should run both. This way the applications on top run on both networks by default. We have too small of a testnet to start partitioning it into two.

**Comment by makeworld-the-better-one** [comment]
@benhylau Won't running both by default possibly run into issues I want to investigate in #243? I feel like making both of them the default for installation could mess some things up until we figure out how it will affect applications. And I'm not in favour of partitioning a network or anything, I mostly made this issue because I think it would be a nice thing to support eventually.

**Comment by darkdrgn2k** [comment]
The applications should be able to resolve this issue by design. Many times there are multiple paths to them (ie have a connection to WiFi AND cell or Ethernet). 

Additionally many of the nodes we use are connected to both Hyperbola and the Internet and that does not create any known issues.

**Comment by darkdrgn2k** [comment]
PS: I encourage you to continue the investigation and see if you can identify any issues. But at present there are no known issues.

**Comment by makeworld-the-better-one** [comment]
Ygg IPFS peering now happens in #268.

**Comment by makeworld-the-better-one** [comment]
There are several [DNS servers on Yggdrasil](https://yggdrasil-network.github.io/services.html#dns) now, including one from @Famicoman.

**Comment by makeworld-the-better-one** [comment]
@ASoTNetworks is running an NTP server at `200:78e8:5817:319d:9cd4:cab3:93ae:6774`.

**Comment by makeworld-the-better-one** [comment]
[In chat](https://matrix.to/#/!DwmKuvGvRKciqyFcxv:matrix.org/$15504208349daCLu:aseriesoftubez.com?via=matrix.org&via=tomesh.net&via=disroot.org) @ASoTNetworks says "Looks good".

**Comment by darkdrgn2k** [comment]
seperate module?

**Comment by makeworld-the-better-one** [comment]
We can now add those services to the stack at least, and make CJDNS a separate module the future possibly.

**Comment by darkdrgn2k** [comment]
right now both services are a sperate module

because  if your not connected to hyperborea they will break the node

same issue will occur with ygg id not peered correctly
